kots-exporter/templates/circle-migrator.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
     app: circle-migrator
     layer: application
 spec:
+  ttlSecondsAfterFinished: 86400   # 24- hrs
```

Comment:
This is neat. The only interesting thing to note is that the feature become first enabled (beta) by default in Kubernetes 1.21 and is now stable in 1.23. Not sure if we care but for customers on k8s < 1.21 it likely won't work unless they have manually enabled alpha features.

Reply:
In that case, we need to provide the deletion instruction as well.

Reply:
For Server 4, [a minimum k8s 1.22 is required.](https://circleci.com/docs/server/installation/phase-1-prerequisites#create-a-kubernetes-cluster)